docs/plugins/gitlab-ce-docker-zh.md

```diff
@@ -0,0 +1,36 @@
+# gitlab-ce-docker 插件
```

Comment:
Please use the `gitlab-ce-docker.zh.md` as a file name. Because this is the normative way to write mkdocs multilingual. 

By the way, please help create a `good first issue` so that the `dtm develop` subcommand supports automatic scaffolding of the `PLUGIN_NAME.zh.md` file

Reply:
Done. I didn't catch this detail at first, thank you!